services/conference/src/scripts/models/audio/StereoManager.ts

```diff
@@ -87,10 +89,8 @@ export class StereoManager {
 
   set audioOutputMuted(muted: boolean) {
     console.log('audioOutputMuted', muted)
-    if (muted) {
-      this.audioElement.pause()
-    }else {
-      this.audioElement.play()
+    for (const id in this.nodes) {
+      this.nodes[id].setMute(muted)
```

Comment:
This setting seems have conflict with individual mute.
For example, we mute our speaker, but in individual A's setting, we unmute A. Then our speaker would play sound from A. That's unexpected behavior.
A better solution would have two settings: global mute and individual mute. We use observe those two settings and change setting of NodeGroup for every individual.

Reply:
This mute is for global mute. When footer's button clicked.
For individual mute remote different system is needed.

Reply:
This mute is for global mute. When footer's button clicked.
For individual mute remote different system is needed.

Reply:
This mute is for global mute. When footer's button clicked.
For individual mute remote different system is needed.

Reply:
This mute is for global mute. When footer's button clicked.
For individual mute remote different system is needed.

Reply:
Yeah, I understand. But when user only want to mute one participant, he have to call someNode.setMute(true).
For example, 
If user 1. mute global, 2. unmute participant A => participant A's sound would be played.
If user 1. unmute participant A, 2. mute global => participant A's sound would not be played.
That's unexpected and unconsistent.
The logic here should be: `A is muted = use mute global || user mute A`

Thus, the mute global API here in StereoManager is just used for stop the global audio output. If you want to mute individual stream, please use api in every NodeGroup.

Reply:
I think mute each participant will be not needed. Too complex and bring many problems to users.